spark-connector/v3.4/spark-runtime/build.gradle.kts

```diff
@@ -13,7 +13,7 @@ plugins {
 val scalaVersion: String = project.properties["scalaVersion"] as? String ?: extra["defaultScalaVersion"].toString()
 val sparkVersion: String = libs.versions.spark34.get()
 val sparkMajorVersion: String = sparkVersion.substringBeforeLast(".")
-val baseName = "${rootProject.name}-spark-connector-runtime-${sparkMajorVersion}_$scalaVersion"
```

Comment:
why remove `${rootProject.name}`?

Reply:
gravitino-spark-connector-runtime seems more meaningful

Reply:
The artifact name for other jars does not contains `gravitino`

Reply:
I perfer to keep `gravitino`,  `spark-connector-runtime` is too general to use in Spark

Reply:
> gravitino-spark-connector-runtime seems more meaningful

So, do we need to add prefix `gravitino` for other project?
<img width="1116" alt="image" src="https://github.com/datastrato/gravitino/assets/15794564/c48156dc-1d0d-4642-b215-f71bc3bfeb8f">

Reply:
Does  artifact name must consistent with the package name? If yes,  I prefer to add `gravitino` to spark connector , if not, I'm ok to the artifact name.

Reply:
no need to keep consistent with other modules like API?

Reply:
This is package will be used for Spark. So it's better to have project like iceberg jar.

Reply:
> Does artifact name must consistent with the package name?

I'm not 100% sure about it, but artifact names and package names are identical are the default behaviour.

Reply:
ok, how about keep gravitino for spark connector, and keep the other modules unchanged?

Reply:
I have not preference. If adding a gravitino prefix is more meaningful, I'm fine with it.